src/test/java/com/moneylendingapp/services/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,64 @@
+package com.moneylendingapp.services.impl;
+
+import com.moneylendingapp.dto.SignUpRequest;
+import com.moneylendingapp.entities.User;
+import com.moneylendingapp.repositories.UserRepository;
+import com.moneylendingapp.services.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Optional;
+import static com.moneylendingapp.enums.EmploymentStatus.CONTRACT;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class UserServiceImplTest {
+
+    @Mock
+    private UserRepository userRepoTest;
+    private UserService userServiceTest;
+
+    @BeforeEach
+    void setUp() {
+        userServiceTest = new UserServiceImpl(userRepoTest);
+    }
+
+
+    @Test
+    void createUser() {
+
```

Comment:
Good job on the test 👏 , we should talk about formatting this together and adding more test cases. It is just one here